portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,27 +15,43 @@
 package com.google.sps.servlets;
 
 import com.google.gson.Gson;
-import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.List;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns a sample list of strings. */
-@WebServlet("/sample_strings")
+/** Servlet that receives user's comment (if any) and returns all users' comments for that session. */
+@WebServlet("/submit_comment")
 public class DataServlet extends HttpServlet {
+  private List<String> comments = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List<String> stringsToShow = Arrays.asList("Lorem ipsum dolor sit amet, consectetur adipiscing elit. ", 
-      "Cras id turpis et nulla semper laoreet. ", 
-      "Nullam gravida consequat risus. ");
-
-    String json = (new Gson()).toJson(stringsToShow);
-
+    String[] commentArr = comments.stream().toArray(String[]::new);
+    String json = (new Gson()).toJson(commentArr);
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = getParameter(request, "comment", "");
```

Comment:
What if you receive an empty parameter? 

Another optional thing is that do we want to perform some input validation? E.g, what if the comment is a series of whitespaces?

Reply:
Haha. Yes, good point! I haven't added these validations intentionally because I just want the basics to work. 😅 I intend to add them later.

Reply:
Please resolve the conversation if the comment has been resolved

Reply:
For the work you intended to add later, please add a TODO here :)

Reply:
Okay!

Reply:
Hmmm I don't see the TODO being added here?

Reply:
I added it to the top of the class 😅 Refer to line 27 because I also want to remind myself to create a class for comment.

Reply:
Oh I mean the comment for input validation

Reply:
Oh I see... Okay!